basic/barrier/src/main/resources/META-INF/spring/integration/client-context.xml

```diff
@@ -14,8 +14,14 @@
 	<int:channel id="requestChannel"/>
 
 	<int-http:outbound-gateway request-channel="requestChannel" 
-					url="http://localhost:8080/receiveGateway"
+					url="#{url}"
 					http-method="POST"
 					expected-response-type="java.lang.String"/>
 
+	<beans profile="default">
```

Comment:
I can merge it as is, but wonder, first of all, what you have had in mind introducing this `profile`-based configuration...